src/i965_drv_video.c

```diff
@@ -6720,6 +6720,7 @@ i965_Terminate(VADriverContextP ctx)
     int i;
 
     if (i965) {
+#if HAVE_HYBRID_CODEC
```

Comment:
That would be good information to put in the commit message and file an issue for.